packages/wallet/src/types.ts

```diff
@@ -21,6 +21,7 @@ export type InitializeTxProps = {
   extraSigners?: TransactionSigner[];
   signingOptions?: SignTransactionOptions;
   scripts?: Cardano.Script[];
+  redeemers?: Cardano.Redeemer[];
```

Comment:
I think we need all of the witness properties (and probably `isValid` too) to be able to compute the correct transaction size for fees. Maybe let's also group it under a 'witness' object (suggestion for both InitializeTxProps and FinalizeTxProps)?
```suggestion
  witness?: {
    datums?: Cardano.Datum[];
    redeemers?: Cardano.Redeemer[];
    bootstrap?: Cardano.BootstrapWitness[];
    extraSigners?: TransactionSigner[];
  };
  isValid?: boolean;
```

Reply:
I understand grouping them under witness, but `InitializeTxProps` does not have all the abovementioned properties. Should I add them?

Reply:
Done here, @AngelCastilloB : [caaa978](https://github.com/input-output-hk/cardano-js-sdk/pull/607/commits/caaa978bf3ab65355bfb85391126e08c14e18d3a).
The original commit message must be changed to include all breaking changes.
Let's keep this comment open until the squash so I don't forget to update it.

Reply:
I updated the commit message with the breaking changes